bin/disco_rds.py

```diff
@@ -55,6 +56,16 @@ def get_parser():
     parser_cleanup_snapshots.set_defaults(mode="cleanup_snapshots")
     parser_cleanup_snapshots.add_argument('--age', dest='days', required=False,
                                           help='Minimum age of Snapshots to expire', type=int, default=30)
+
+    # clone Mode
+    parser_clone = subparsers.add_parser("clone", help="Create a new database from an existing database")
+    parser_clone.set_defaults(mode="clone")
+    parser_clone.add_argument('--env', dest='env', required=False, default=None,
```

Comment:
Side note on this: disco_rds behaves differently from all other multi-command scripts (AFAIK), in that it takes the env as an argument to the subcommand rather than to the main command.  Worth mucking with to improve consistency?